ops/demo/main.tf

```diff
@@ -113,15 +113,15 @@ module "nat_gateway" {
   tags                    = local.management_tags
 }
 
-/* module "web_application_firewall" {
+module "web_application_firewall" {
   source                  = "../services/web_application_firewall"
   name                    = local.name
   env                     = local.env
   resource_group_location = data.azurerm_resource_group.rg.location
   resource_group_name     = data.azurerm_resource_group.rg.name
 
   tags = local.management_tags
-} */
+}
```

Comment:
Does `firewall_policy_id` need to be set here too? You [mentioned](https://github.com/CDCgov/prime-simplereport/pull/3919/files#r911367962) it's already present on `dev` and `prod`, not sure about `demo` though.

Reply:
Excellent catch, thank you for this! This was just fixed.